openshift-client/README.md

```diff
@@ -1,118 +1,77 @@
-    
-    ```
-    oc apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/openshift-client/taskUsingServiceaccount/ocpipelinerun.yaml 
-    ```
-
-5. Check the logs of the pipeline by accessing the logs of a freshly created pods(Each task have one Pod).
-
-## Using oc on a different cluster using clusterResource
-
-You can use the clusterResource to interact on a different cluster with `oc` as follows: 
-
-1. Create a [clusterResource](https://github.com/tektoncd/pipeline/blob/master/docs/resources.md#cluster-resource) with details for the second cluster. For example, here is a [sample ClusterResource](https://raw.githubusercontent.com/tektoncd/catalog/master/openshift-client/taskUsingClusterResource/resource.yaml):
-
-    ```yaml
-    apiVersion: tekton.dev/v1alpha1
-    kind: PipelineResource
-    metadata:
-      name: cluster-details
-    spec:
-      type: cluster
-      params:
-      - name: url
-        value: cluster url
-      - name: name
-        value: oc-test
-      - name: username
-        value: test
-      - name: token
-        value: akakkakaaa
-      - name: cadata
-        value: data
-    ```
-
-        
-2. Create a task which will used for executing every command
-    
-    ```
-    oc apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/openshift-client/openshift-client-kubecfg-task.yaml
-    ```
- 
-3. Create a pipeline which have the tasks used to interact with the OpenShift Cluster
-    
-    ```
-    oc apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/openshift-client/taskUsingClusterResource/ocpipeline.yaml 
-    ```
-
-4. Create a pipelinerun to execute the pipeline you created. Make sure that the ClusterResource Name is used correctly in the pipelinerun.
-
-    ```
-    oc apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/openshift-client/taskUsingClusterResource/ocpipelinerun.yaml 
-    ```
-
-5. Check the logs of the pipelinerun by accessing the logs of the freshly created pods(Each task have one Pod).
\ No newline at end of file
+* **ARGS:** args to execute which are appended to `oc` e.g. `start-build myapp` (_default_: `help`)
+
+### Resources
+
+* **cluster**: a `cluster`-type `PipelineResource` specifying the target OpenShift cluster to execute the commands against it (only for `openshift-client-kubecfg` task) 
+
+## ServiceAccount
+
+If you don't specify a service account to be used for running the `TaskRun` or `PipelineRun`, the `default` [service account](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#use-the-default-service-account-to-access-the-api-server). OpenShift by default does not allow the default service account to modify objects in the namespace. Therefore you should either explicitly grant permission to the default service account (by creating rolebindings) or create a new service account with sufficient privileges and specify it on the `TaskRun` or `PipelineRun`.
```

Comment:
Shouldn't we document both (aka creating a specific service account or editing the default one), and recomend for the specific one ?

Reply:
I think it's enough to point to the right direction. Added links to Tekton and Kubernetes docs.